src/furtherWork/petGmoves.js

```diff
@@ -0,0 +1,14 @@
+const {
+    damageValues,
+    attackMoves
+} = require("./furtherWork/gameProperties.js");
```

Comment:
This file path doesnt exist so is causing the test to not be loaded, it should be
```
} = require("./gameProperties.js");
```